content/en-us/luau/strings.md

```diff
@@ -80,10 +80,10 @@ To convert a string to a number, use the `Global.LuaGlobals.tonumber()` function
 
 ```lua
 local numericString = "123"
-print(tonumber(numericString))  --> 123
+print(tonumber(numericString)) --> 123
 
 local alphanumericString = "Hello123"
-print(tonumber(alphanumericString))  --> nil
+print(tonumber(alphanumericString)) --> nil
 ```
 
 ## Escaping Strings
```

Comment:
The script made too many assumptions in this case, please revert. It's about using backslash to embed double quotes, not using single quotes as the wrapper instead.